extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -9,6 +9,8 @@ const errors = require('arsenal').errors;
 const BackbeatConsumer = require('../../../lib/BackbeatConsumer');
 const FailedCRRProducer =
     require('../../../extensions/replication/failedCRR/FailedCRRProducer');
```

Comment:
Maybe the file was moved in the past? The path here can be shortened

Reply:
Yeah, I can silent fix this.